runtime/gc_include/ObjectAccessBarrierAPI.hpp

```diff
@@ -2625,7 +2625,8 @@ class MM_ObjectAccessBarrierAPI
 	VMINLINE void
 	internalPreStoreObject(J9VMThread *vmThread, j9object_t object, fj9object_t *destAddress, j9object_t value)
 	{
-		if (j9gc_modron_wrtbar_realtime == _writeBarrierType) {
+		if ((j9gc_modron_wrtbar_satb == _writeBarrierType) ||
+				(j9gc_modron_wrtbar_satb_and_oldcheck == _writeBarrierType)) {
 			internalPreStoreObjectRealtime(vmThread, object, destAddress, value);
```

Comment:
rename internalPreStoreObjectRealtime to internalPreStoreObjectSATB

Reply:
renamed

Reply:
oh right, renamed both now